Sam edits

sammurraytuesta opened the pull request:
Update Webpage and add content like blog posts, images, and about sections to add personality to the site!

rjkroege commented on portfolio/src/main/webapp/script.js:
```diff
@@ -15,14 +15,17 @@
+  const facts =
+      ['I play the cello!', '¡Soy peruana y americana!', 'I love Marvel!', '¡Me encanta dibujar!', 'I am from Seattle!', 'I love watching basketball!'];
 
   // Pick a random greeting.
```
Change greeting to fact? Comments and code should be in sync.

sammurraytuesta replied:
Greeting is now changed to fact. This js file should now be consistent!

rjkroege commented on portfolio/src/main/webapp/style.css:
```diff
@@ -7,3 +7,7 @@
+
+.main-image{
+    width: 650px;
+}
```
Try to end files with a newline.

sammurraytuesta replied:
Noted! Thank you!

rjkroege commented on portfolio/src/main/webapp/index.html:
```diff
@@ -10,14 +10,60 @@
       <div id="greeting-container"></div>
+      <img class="profile-pic" src="images/profilePicture.JPG"/>      
+      <h3>About Me</h3>
+      <p>I am a first generation college student, in my second year studying Computer Science and Internet Studies. I am currently based in Seattle, WA, and attend college in Bellingham!
```
Please line break text inside block elements.

sammurraytuesta replied:
Elements are now spaced accordingly!

rjkroege commented on portfolio/src/main/webapp/index.html:
```diff
@@ -10,14 +10,60 @@
+      <img class="profile-pic" src="images/profilePicture.JPG"/>      
+      <h3>About Me</h3>
+      <p>I am a first generation college student, in my second year studying Computer Science and Internet Studies. I am currently based in Seattle, WA, and attend college in Bellingham!
+         <br><br>At my institution, Western Washington University, I am engaged in research specializing in digital privacy and its portrayal in the media. My work is aimed at creating a digital web game to engage students of all ages and teach the importance of digital privacy and how we go about advocating for those impacted by breaches in that privacy.
```
You are using `<br>` elements to control the visual appearance? I think that this runs contrary to the aspirational goal of HTML/CSS in that tags should express document semantic structure while CSS should control how it's rendered to the page. Consider using a `<p>` element for each block and having a CSS class that controls spacing?

sammurraytuesta replied:
The HTML has now been changed to <p> elements with CSS styling to adjust its proximity to other elements on the page!

rjkroege commented on portfolio/src/main/webapp/index.html:
```diff
@@ -10,14 +10,60 @@
+      </p>
+      <h3>Projects</h3>
+      <ul>
+          <li>
```
This seems like a bit of an abuse of the intent of the `<ul>` tag to me. We can discuss later.

sammurraytuesta replied:
Yes I agree! I have changed this to normal paragraph blocks, and incorporated the use of the ul tag in listing off my socials!

rjkroege commented on portfolio/src/main/webapp/index.html:
```diff
@@ -59,11 +59,19 @@ <h3>Socials</h3>
       <p>GitLab: <a href="https://gitlab.com/sammurraytuesta">https://gitlab.com/sammurraytuesta</a></p>
       <h3>Gallery</h3>
       <img class="image" src="images/mini-me-hose.jpg"/>
```
`<figure>` and `<figcaption>` are specifically intended for this use case of having an image and a associated caption. Consider using them instead.

sammurraytuesta replied:
Thank you! I have made that change and learned something new!

rjkroege reviewed (changes requested):
Why did you add the carousel to your in-progress PR? I would like you to:

* finish your previous PR by responding to my comments
* merge that
* and then create a new PR for adding the carousel

Also: did you need to check-in all of owl? Are there smaller choices? Can you add the third-party code (owl) in a separate PR from the code that uses it?

sammurraytuesta commented on portfolio/src/main/webapp/script.js:
```diff
@@ -15,14 +15,17 @@
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFunFact() {
+  // adjusted from greetings to facts about me!
```
Comment is now added to the doc comment above!

sammurraytuesta commented on portfolio/src/main/webapp/script.js:
```diff
@@ -15,14 +15,17 @@
+function addFunFact() {
+  // adjusted from greetings to facts about me!
+  // functionality is the same as addRandomGreeting(),
+  // so content in array is replaced while the basic functionality remains the same
```
Perfect! Thank you! I have changed all comments to complete sentences! Let me know if I missed any! Thank you!

sammurraytuesta commented on portfolio/src/main/webapp/script.js:
```diff
@@ -15,14 +15,17 @@
+  // functionality is the same as addRandomGreeting(),
+  // so content in array is replaced while the basic functionality remains the same
+  const facts =
+      ['I play the cello!', '¡Soy peruana y americana!', 'I love Marvel!', '¡Me encanta dibujar!', 'I am from Seattle!', 'I love watching basketball!'];
```
Thank you! :)

sammurraytuesta commented on portfolio/src/main/webapp/js/lightbox.js:
```diff
@@ -0,0 +1,4 @@
+
+$('document').ready(function () {
+    baguetteBox.run('.baguetteBoxOne');
+});
```
This will be updated in the design-edits PR! Thank you!

sammurraytuesta commented:
I have addressed all requested changes in my most recent commit to sam-edits and moved the new design choices and owl.carousel code to their own branches to later be submitted for PRs. As of right now, merging is still blocked and it still says there is one change requested, does that mean I should re-request review or is there something that I am missing? Thank you!

rjkroege commented:
I was hoping that you'd separate the review response to your first PR and make a new PR for adding OWL support. This PR still adds a lot of OWL-related functionality. Can you divide this up more:

* *just* the review responses from the first PR
* a different PR where you add the carousel library
* a different PR where you modify your code to take advantage of the carousel?

sammurraytuesta commented on portfolio/src/main/webapp/css/style.css:
```diff
@@ -0,0 +1,181 @@
+}
+
+.profile-pic{
+    fill: url(#pattern);
```
yeah! So basically the image has the id pattern and I substitute the image in as the url for it to populate the inner area of the svg line!

sammurraytuesta commented on portfolio/src/main/webapp/css/style.css:
```diff
@@ -0,0 +1,181 @@
+}
+
+
+.owl-carousel{
```
I see what you are saying! Thank you!

sammurraytuesta commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,77 +1,194 @@
+  </div>
+
+  <section id="home">
+    <svg class="home-blob" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
```
I used a website known as svg blob generator to generate an svg line based on the shape I created using the tools on the page!
